src/test/resources/org/broadinstitute/hellbender/tools/picard/analysis/CollectInsertSizeMetrics/expectedQualityYieldOnInsertSizeMetrics.txt

```diff
@@ -0,0 +1,10 @@
+## htsjdk.samtools.metrics.StringHeader
+# org.broadinstitute.hellbender.tools.spark.pipelines.metrics.CollectMultipleMetricsSpark  --output /var/folders/3g/0dh8mspj3pz8qpmm4qqqzblr0000gn/T/collectMultiMetricsTest381676565295870422/collectMultiMetrics --collectors CollectInsertSizeMetrics --collectors CollectQualityYieldMetrics --input /Users/cmn/projects/hellbender/src/test/resources/org/broadinstitute/hellbender/tools/picard/analysis/CollectInsertSizeMetrics/insert_size_metrics_test.sam --verbosity ERROR  --assumeSorted true --metricAccumulationLevel ALL_READS --readValidationStringency SILENT --interval_set_rule UNION --interval_padding 0 --bamPartitionSize 0 --disableSequenceDictionaryValidation false --shardedOutput false --numReducers 0 --sparkMaster local[*] --help false --version false --QUIET false
```

Comment:
this file was clearly generated using hellbender test suite. is it the same as a file generated using picard on the same inputs? I think we should (whenever possible) use expected results that were created from picard (or gatk3 when appropriate)